### PR TITLE
Roadmap and ideas

### PR DESCRIPTION
**Pull request by kumare3**
Updated the docs to have a roadmap section, also created a roadmap Spreadsheet and a ideas document

### CODE REVIEW

File: rsts/introduction/roadmap.rst [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+3. Observability
+----------------
+   We are currently working on an one time Observability epic that will enable all our open source users to use dashboard templates exactly like we use within Lyft (as of April 2020). Going forward,
+   Observability is one of the pillars of Flyte. We will continue emphasizing on this aspect and improving it with every iteration.
+
+
+Living Roadmap
+===============
+We are trying to maintain a Living roadmap `here <https://docs.google.com/spreadsheets/d/1V8DQfcsX_02Zac5EfAo0UrGJtLwdMPcw3wuuigVIMZU/edit?usp=sharing>`_
```
Comment (wild-endeavor) [inline rsts/introduction/roadmap.rst:41]: Perhaps some treatment on how the open source community can voice their opinion and get things added to the roadmap?  Should they open GitHub issues?  Will they be required to defend their ideas in the weekly meetings?

File: rsts/introduction/roadmap.rst [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+   effort will continue on an ongoing basis and we would love feedback on the top areas where we should improve.
+
+2. Bug Fixes & Reliability Improvements
+----------------------------------------   
+   As mentioned since Flyte is being actively used at Lyft and our scale is always increasing, there are always bugs that we want to squish. So Bugs, are always work in progress and most often then
+   not receive highest priority.
+
+3. Observability
+----------------
+   We are currently working on an one time Observability epic that will enable all our open source users to use dashboard templates exactly like we use within Lyft (as of April 2020). Going forward,
```
Comment (wild-endeavor) [inline rsts/introduction/roadmap.rst:35]:
```suggestion
   We are currently working on a one time Observability epic that will enable open source users to use dashboard templates exactly like we use within Lyft (as of April 2020). Going forward,
```

File: rsts/introduction/roadmap.rst [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+This section highlights features and issues that are a continuous area of Focus and will always be in progress. 
+
+1. Documentation 
+----------------
+   Flyte's AIM is to make it absolutely easy for any new user, administrator, contributor to read the documentation and complete their job successfully. We realize That we are not even close. This
+   effort will continue on an ongoing basis and we would love feedback on the top areas where we should improve.
+
+2. Bug Fixes & Reliability Improvements
+----------------------------------------   
+   As mentioned since Flyte is being actively used at Lyft and our scale is always increasing, there are always bugs that we want to squish. So Bugs, are always work in progress and most often then
```
Comment (wild-endeavor) [inline rsts/introduction/roadmap.rst:30]: This makes it feel like bugs are a low priority, which isn't always the case depending on the severity of the bug.

File: rsts/introduction/roadmap.rst (hunk 1 above)
Comment (kumare3) [reply rsts/introduction/roadmap.rst:41]: agreed, lets add something

File: rsts/introduction/roadmap.rst (hunk 3 above)
Comment (kumare3) [reply rsts/introduction/roadmap.rst:30]: really, I feel this says bugs are found and fixed continuously, but maybe we need to create a process of ranking. I do not want to say we will get to them ASAP, as we have not done that

**Pull request merged**